### ISSUE TITLE
AutoML accuracy differs from Pipeline Profiler

### ISSUE THREAD

**Issue by enadio** [VIDA-NYU/PipelineVis#5]
Gents,

No sure if this is an issue at all. But I hardly get the same accuracy results when comparing autosklearn.classification.AutoSklearnClassifier() and the respective Pipeline Profiler (PP) plot. It normally show autosklearn wtih a much higher accuracy than PP. If necessary, I could send more info.

Tks.

**Comment by jorgehpo** [comment]
Hi @enadio, thanks for reaching out.

The difference you are seeing is because autosklearn creates an ensemble classifier from all the pipelines tested. If you switch the metric to Ensemble Weights, you can check out how these pipelines are combined (we have an example here https://towardsdatascience.com/exploring-auto-sklearn-models-with-pipelineprofiler-5b2c54136044 - fig 2)

**Comment by enadio** [comment]
Thanks for the reply.

I am running Profiler, but when I select the option ensemble weight, It
does not show  new information on the left side.

Em sex., 18 de dez. de 2020 às 13:16, JorgeHPO <notifications@github.com>
escreveu:

**Comment by jorgehpo** [comment]
Hi @enadio , you should see new information on the right side (the metrics view, right below the metric drop down select).
